models/migrations/v35.go

```diff
@@ -10,12 +10,18 @@ import (
 	"github.com/go-xorm/xorm"
 )
 
-func addLoginSourceLdapPublicSSHKeySyncEnabled(x *xorm.Engine) error {
-	// LoginSource see models/login_source.go
+func NewPublicKeyandLoginSourceColumns(x *xorm.Engine) (err error) {
```

Comment:
Add comment `// NewPublicKeyandLoginSourceColumns ...` otherwise `make lint` is failing